prac_06/programming_language.py

```diff
@@ -0,0 +1,21 @@
+"""Class defining a Programming Language."""
+
+
+class ProgrammingLanguage:
+    """Represents a Programming Language object."""
+
+    def __init__(self, name, typing, reflection, year_created):
+        """Initialise a Programming Language instance."""
+        self.name = name
+        self.typing = typing
+        self.reflection = bool(reflection)
+        self.year_created = int(year_created)
```

Comment:
You could simplify this line by writing it as self.year_created = year_created.

Reply:
Same as above.